pandas/plotting/_core.py

```diff
@@ -663,6 +678,13 @@ class PlotAccessor(PandasObject):
         detail.
     xerr : DataFrame, Series, array-like, dict and str
         Equivalent to yerr.
+    stacked : bool, default False in line and bar plots, and True in area plot
+        If True, create stacked plot.
+    sort_columns : bool, default False
+        Sort columns names to determine plot ordering.
```

Comment:
Not sure, but shouldn't be "column names"?

Reply:
that's right, column names,  should have checked for all these inaccuracies
thanks for your time!
I'll make changes shortly